.github/workflows/CI_push_pypi.yml

```
@@ -0,0 +1,40 @@
+name: Pypi Build
+
+on: [push]
+
+jobs:
+
+  build:
+    name: Build for (${{ matrix.python-version }}, ${{ matrix.os }})
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ['ubuntu-latest', 'windows-latest']
+        python-version: ['3.7', '3.8']
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Python info
+        shell: bash -l {0}
+        run: |
+          which python
+          python --version
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+      - name: Build
+        shell: bash -l {0}
+        run: |
+          python setup.py build
+      - name: Show pip list
+        shell: bash -l {0}
+        run: |
+          pip list
+      - name: Test
+        shell: bash -l {0}
+        run: |
+          python setup.py test
```

Using `setup.py test` command is [deprecated](https://setuptools.readthedocs.io/en/latest/userguide/commands.html?highlight=test#test-build-package-and-run-a-unittest-suite) use pytest directly

```suggestion
          pytest
```

---

Thanks, didn't know that.